### PR TITLE
[Ready] Adds new huds that help correct vision.

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Some Code origination  with tools now being in a labeled tools section 
Adds 2 new nodes for all the new goggles and huds with correction lens
A counter part to each type of HUDs that are ***NOT NV*** with a prescription lens
A brand new cargo crate that has 2 Sunglasses that are also sec huds for 2500
Miner gar glasses now are prescription lens do to Power Creep
[why]
QoL and basic things for people that have neg traits and get fucked over by it. 
After all we have NV prescription goggles lets branch it out!
Code origination for Qol for code diving.

### CODE REVIEW

File: code/modules/clothing/glasses/hud.dm [hunk 1]
```diff
@@ -116,6 +140,16 @@
 	tint = 1
 	glass_colour_type = /datum/client_colour/glass_colour/darkred
 
+/obj/item/clothing/glasses/hud/security/sunglasses/prescription
```
Comment (deathride58) [inline code/modules/clothing/glasses/hud.dm]: No.

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -322,6 +322,15 @@
 					/obj/item/gun/energy/laser)
 	crate_name = "laser crate"
 
+/datum/supply_pack/security/prescription_sec_sung
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: No.

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by JogBoba** [comment]
I'd rather see these on the same node that unlocks the original item, as nightvision does. Or on the Nightvision node itself, as the added nodes bloat points.

File: code/modules/clothing/glasses/hud.dm [hunk 3]
```diff
@@ -37,6 +37,14 @@
 	hud_type = DATA_HUD_MEDICAL_ADVANCED
 	glass_colour_type = /datum/client_colour/glass_colour/lightblue
 
+/obj/item/clothing/glasses/hud/health_prescription
```
Comment (deathride58) [inline code/modules/clothing/glasses/hud.dm]: Make this a subtype of `/obj/item/clothing/glasses/hud/health`

File: code/modules/clothing/glasses/hud.dm [hunk 4]
```diff
@@ -62,6 +70,14 @@
 	hud_type = DATA_HUD_DIAGNOSTIC_BASIC
 	glass_colour_type = /datum/client_colour/glass_colour/lightorange
 
+/obj/item/clothing/glasses/hud/diagnostic_prescription
```
Comment (deathride58) [inline code/modules/clothing/glasses/hud.dm]: Make this a subtype of `/obj/item/clothing/glasses/hud/diagnostic`

File: code/modules/clothing/glasses/hud.dm [hunk 5]
```diff
@@ -78,6 +94,14 @@
 	hud_type = DATA_HUD_SECURITY_ADVANCED
 	glass_colour_type = /datum/client_colour/glass_colour/red
 
+/obj/item/clothing/glasses/hud/security_prescription
```
Comment (deathride58) [inline code/modules/clothing/glasses/hud.dm]: Make this a subtype of `/obj/item/clothing/glasses/hud/security`

**Review by deathride58 — DISMISSED** [review]

**Review by silicons — APPROVED** [review]

**Pull request merged**